blueman/gui/Notification.py

```diff
@@ -167,10 +167,18 @@ class Notification(object):
     def actions_supported():
         return "actions" in Notify.get_server_caps()
 
+    @staticmethod
+    def daemon_available():
+        return Notify.get_server_caps() != []
```

Comment:
The api is supposed to return a list which may be empty, see [docs](https://developer.gnome.org/libnotify/0.7/libnotify-notify.html#notify-get-server-caps). But I agree checking for ``body`` is more clean. Updated the commit, thx.